### PR TITLE
feat: add the custom colors

### PR DESCRIPTION
**Pull request by christelle-gihozo**
I added two custom colors

### CODE REVIEW

**Review by Kevin-gram — COMMENTED** [review]
Don't you think that the name of this green is longer? I think we can call it PrimGreen or any other short name we can go along with!

File: tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,12 @@
 module.exports = {
   content: ["./*.{html,js}"],
   theme: {
-    extend: {},
+    extend: {
+      colors:{
+        ourPrimaryGreen: "#00672E",
+        bgPrimary: "EFECEA"
```
Comment (Gasorekibo) [inline tailwind.config.js]: You may have forgotten to add # in front of the hexadecimal color.

Comment (christelle-gihozo) [reply tailwind.config.js]: Thank you, I have added the #

**Review by brendahuwitonze — APPROVED** [review]
Perfect

**Review by Gasorekibo — APPROVED** [review]

**Comment by christelle-gihozo** [comment]
perfect!!!

**Pull request closed without merging**